notify/notification_apns_test.go

```diff
@@ -83,7 +83,7 @@ func TestIOSNotificationStructure(t *testing.T) {
 		URLArgs:  []string{"a", "b"},
 	}
 
-	notification := GetIOSNotification(req)
+	notification, _ := GetIOSNotification(req)
```

Comment:
please, add new tests for `err != nil` case

Reply:
I think just because failing to marshal the json shouldn't block the notification, I will modify the usages and only log as error.

Reply:
As I mentioned before; 
just because **log** parser error, we don't have to return err parameter, so I removed it from return statement